lib/printer.js

```diff
@@ -954,6 +1006,9 @@ function genericPrintNoParens(path, options, print) {
     case "Specifier":
     case "NamedSpecifier":
     case "Comment": // Supertype of Block and Line.
+    case "MemberTypeAnnotation": // Flow
+    case "TupleTypeAnnotation": // Flow
+    case "Type": // Flow
```

Comment:
Member type is deprecated. Type is abstract. Tuple type is

    var x:[number, string] = [123, "hi"];